microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileValidator.java

```diff
@@ -80,7 +80,7 @@ private void validateProperty(Property property) {
 		String propertyNameWithProfile = property.getPropertyNameWithProfile();
 		if (propertyNameWithProfile != null && !propertyNameWithProfile.isEmpty()) {
 			// Validate Syntax property

```

Comment:
Remove this comment; the function name describes what the function does better.

Reply:
thank you, fixed